CONTRIBUTING.md

```diff
@@ -278,7 +286,7 @@ An entry in `bin` needs to be in `PATH` identify with `which external_dependency
 An entry in `python` needs to be in `PYTHONPATH` identify with `python -c "import external_dependency_python_N"` command.
 
 #### ImportError
-In python files where you use a `import external_dependency_python_N` you will need to add a `try-except` with a info log.
+In python files where you use a `import external_dependency_python_N` you will need to add a `try-except` with a debug log.
```

Comment:
Can you reference this change in this PR to the discussion we had on it? It might deserve it's own PR.